Submitted By: Bruce Dubbs <bdubbs@linuxfromscratch.org>
Date: 2013-03-08
Initial Package Version: 1.2.0
Upstream Status: Patch has been submitted
Origin: qemu mailing list
Description: Fix texi file error exposed by Texinfo-5.0

diff -Naur qemu-kvm-1.2.0/qemu-options.hx qemu-kvm-1.2.0.mod/qemu-options.hx
--- qemu-kvm-1.2.0/qemu-options.hx	2012-09-06 03:31:27.000000000 -0500
+++ qemu-kvm-1.2.0.mod/qemu-options.hx	2013-03-08 15:35:56.000000000 -0600
@@ -1964,18 +1964,13 @@
 devices.
 
 Syntax for specifying a sheepdog device
-@table @list
-``sheepdog:<vdiname>''
-
-``sheepdog:<vdiname>:<snapid>''
-
-``sheepdog:<vdiname>:<tag>''
-
-``sheepdog:<host>:<port>:<vdiname>''
-
-``sheepdog:<host>:<port>:<vdiname>:<snapid>''
-
-``sheepdog:<host>:<port>:<vdiname>:<tag>''
+@table @code
+@item sheepdog:<vdiname>
+@item sheepdog:<vdiname>:<snapid>
+@item sheepdog:<vdiname>:<tag>
+@item sheepdog:<host>:<port>:<vdiname>
+@item sheepdog:<host>:<port>:<vdiname>:<snapid>
+@item sheepdog:<host>:<port>:<vdiname>:<tag>
 @end table
 
 Example
